clients/admin-ui/src/features/system/form.ts

```diff
@@ -91,4 +96,12 @@ export const transformFormValuesToSystem = (formValues: FormValues): System => {
         ? undefined
         : formValues.administrating_department,
   };
+  if (features.plus) {
+    if (features.dictionaryService && formValues?.meta?.vendor?.id) {
```

Comment:
I broke it up it to two nested ifs just in case we other additional plus only features that could be toggled on or off